modules/localgov_services_status/localgov_services_status.module

```diff
@@ -37,6 +38,63 @@ function localgov_services_status_theme($existing, $type, $theme, $path) {
   ];
 }
 
+/**
+ * Implements hook_localgov_roles_default().
+ */
+function localgov_services_status_localgov_roles_default(): array {
+
+  // Content editing permissions.
+  $perms = [
+    RolesHelper::EDITOR_ROLE => [
+      'create localgov_services_status content',
+      'delete any localgov_services_status content',
+      'delete localgov_services_status revisions',
+      'delete own localgov_services_status content',
+      'edit any localgov_services_status content',
+      'edit own localgov_services_status content',
+      'revert localgov_services_status revisions',
+      'view localgov_services_status revisions',
+    ],
+    RolesHelper::AUTHOR_ROLE => [
+      'create localgov_services_status content',
+      'delete own localgov_services_status content',
+      'edit any localgov_services_status content',
+      'edit own localgov_services_status content',
+      'revert localgov_services_status revisions',
+      'view localgov_services_status revisions',
+    ],
+    RolesHelper::CONTRIBUTOR_ROLE => [
+      'create localgov_services_status content',
+      'delete own localgov_services_status content',
+      'edit any localgov_services_status content',
+      'edit own localgov_services_status content',
+      'revert localgov_services_status revisions',
+      'view localgov_services_status revisions',
+    ],
+  ];
```

Comment:
Should this block not also be wrapped in `if (\Drupal::moduleHandler()->moduleExists('localgov_roles')) {` ?

Is including localgov_roles as an explicit dependacy now aggreed?  BHCC already has content roles and would rather not have to go and then delete them just becuase we use the service modules. Also would these break if BHCC did not have the defined content editor roles.

Reply:
This code is called by the hook_localgov_roles_default() hook, which is part of localgov_roles. So these permissions only get applied if localgov_roles is enabled.

Reply:
Ok, that makes sense thanks.
So to confirm, if BHCC does not use localgov_roles (yet) that is ok?

Reply:
The `use RolesHelper` statement is still parsed though.

Even if it's in the default code base I never tested if Drupal's autoloader compiler includes disabled modules, so used the fully pathed namesapce (and added a coding standards ignore). If there's a chance it's not in the codebase as it's not used then even more so.

Reply:
> So to confirm, if BHCC does not use localgov_roles (yet) that is ok?

Yep - enabling localgov_roles and then enabling these modules or localgov_workflows will bring in all the permissions, but if you don't have roles enabled you won't get the permissions set